index.html

```diff
@@ -1,26 +1,93 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<title>Homework Assignment Three</title>
     	<meta charset="utf-8">
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
-
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
 
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
-			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
+		<div class="topbar clearfix">
+		<br>
+			<div class="leftfloat">Relaxr</div>
+			<div class="middlefloat"></div>
+			<div class="rightfloat alignright">About FAQ Team Contact Us Blog</div>
 		</div><!-- #container -->
+		<br>
+		<br>
+		<section id="container" class="clearfix">
+			<div>
```

Comment:
The `clearfix` should be added to this `div` since it is the parent of the two floated elements. the `section` would be considered the grandparent